src/llmq/ehf_signals.cpp

```diff
@@ -108,11 +108,11 @@ MessageProcessingResult CEHFSignalsHandler::HandleNewRecoveredSig(const CRecover
     }
 
     const auto ehfSignals = mnhfman.GetSignalsStage(WITH_LOCK(cs_main, return chainstate.m_chain.Tip()));
-    MNHFTxPayload mnhfPayload;
     for (const auto& deployment : Params().GetConsensus().vDeployments) {
         // skip deployments that do not use dip0023 or that have already been mined
         if (!deployment.useEHF || ehfSignals.find(deployment.bit) != ehfSignals.end()) continue;
 
+        MNHFTxPayload mnhfPayload;
```

Comment:
595c4cd88b0935e17ffcdcd096a180d444b48cc0 this commit shouldn't be included here? If it's a fix? I'm confused by it